### PR TITLE
Make all of rooms use sets rather than lists

### PR DESCRIPTION
**Pull request by tnix100**
Some parts of the code for rooms used lists rather than sets.

### CODE REVIEW

File: cloudlink/cloudlink.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def __init__(self):
 
     def server(self, logs=False):
         # Initialize Cloudlink server
-        from .server import server
+        from server import server
```
Comment (tnix100) [inline cloudlink/cloudlink.py:21]: this got fixed in a later commit, github being dumb and not showing it

File: cloudlink/server.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import sys
 import logging
-from .websocket_server import WebsocketServer
-from .supporter import supporter
-from .serverRootHandlers import serverRootHandlers
-from .serverInternalHandlers import serverInternalHandlers
+from websocket_server import WebsocketServer
```
Comment (tnix100) [inline cloudlink/server.py:3]: this got fixed in a later commit, github being dumb and not showing it

File: cloudlink/cloudlink.py (hunk 1 above)
Comment (MikeDev101) [reply cloudlink/cloudlink.py:21]: This breaks importing as a module, which is the intended functionality of CL.

`>>> from cloudlink import Cloudlink`
`>>> cl = Cloudlink()`
`>>> server = cl.server(logs=True)`
`Traceback (most recent call last):`
`  File "<stdin>", line 1, in <module>`
`  File "C:\Users\mikie\Documents\GitHub\package\cloudlink\cloudlink.py", line 21, in server`
`    from server import server`
`ModuleNotFoundError: No module named 'server'`

File: cloudlink/server.py (hunk 2 above)
Comment (MikeDev101) [reply cloudlink/server.py:3]: This breaks importing as a module, which is the intended functionality of CL.

`>>> from cloudlink import Cloudlink`
`>>> cl = Cloudlink()`
`>>> server = cl.server(logs=True)`
`Traceback (most recent call last):`
`  File "<stdin>", line 1, in <module>`
`  File "C:\Users\mikie\Documents\GitHub\package\cloudlink\cloudlink.py", line 21, in server`
`    from .server import server`
`  File "C:\Users\mikie\Documents\GitHub\package\cloudlink\server.py", line 3, in <module>`
`    from websocket_server import WebsocketServer`
`ModuleNotFoundError: No module named 'websocket_server'`

Comment (MikeDev101) [reply cloudlink/server.py:3]: Please stop removing these relative imports. I intended CL to be imported entirely as a module from external code, or imported as a package from PyPi.

**Review by MikeDev101 — APPROVED** [review]

**Pull request merged**